lib/logger/README.md

```diff
@@ -0,0 +1,57 @@
+# logger
+
+The `logger` module provides an object that wraps various console logging methods safely. The default level is INFO, but it is recommended that you set this according to your environment.
+
+## Usage
+
+```javascript
+var Logger = require('bv-ui-core/lib/logger');
+// <method> is one of the 5 types, ex: log
+Logger.<method>('Hello, world!') // console.log('Hello, world!')
+```
+
+### #setLogLevel
+
+Set the log level to one of the following to prevent logging under the specified level.
+```
+Logger.DEBUG
+Logger.LOG
+Logger.INFO
+Logger.WARN
+Logger.ERROR
+Logger.OFF
+```
+
+```javascript
+var Logger = require('bv-ui-core/lib/logger');
+
+// Set's the default log level to one of the valid levels
+// DEBUG, LOG, INFO, WARN, ERROR, OFF
+Logger.setLogLevel(Logger.DEBUG)
+
+Logger.log('Hello, World!') // Prints Hello, World!
+
+Logger.setLogLevel(Logger.INFO)
+
+Logger.log('Hello, World!') // Prints nothing because of logLevel
+```
+
+### group/groupEnd
+https://developer.mozilla.org/en-US/docs/Web/API/console/group
+
+### time/timeEnd
+https://developer.mozilla.org/en-US/docs/Web/API/console/time
+
+### assert
+
+Assertions can be made if `logLevel` is not `Logger.OFF`. You can assert a function's result or a boolean value.
```

Comment:
Does this not? https://github.com/bazaarvoice/bv-ui-core/pull/32/files#diff-f8f0b21002d80d85a7256fac15628da4R32

If a page has the specified cookie present it will automatically set the level. That of course can be overwritten later by anyone calling Logger.setLogLevel(). Do we want the cookie to take precedence over whatever the code is doing?